db/schema.rb

```diff
@@ -56,6 +57,7 @@
     t.string   "question"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.integer  "order"
```

Comment:
And yeah I initially tried to add things manually to the schema file because when I tried to do it with a migration it wasn't populating a data type. Like I would get something like "t.      'order'". But I don't think I was using the correct data type in the migration so I can revert the schema file and try again with a migration. I'm assuming that modifying the schema file is bad form because it doesn't track changes to the database structure the way migrations do?